src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (firstNum) => {
+    count++;
+
+    return count > 3 && count % 2 === 0
+      ? () => 'Bzzz... Error!'
+      : secondNum => firstNum + secondNum;
+  };
+
+  return getSum;
```

Comment:
you can also immediately return the function without saving it in the variable

Reply:
it's a habit) 
helps me to see what exactly the function returns at the end